device/device.go

```diff
@@ -0,0 +1,110 @@
+package device
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/device/deviceType"
+	"github.com/josephroberts/edge-node-manager/radio/bluetooth"
+	"github.com/josephroberts/edge-node-manager/radio/radioType"
+	"github.com/josephroberts/edge-node-manager/radio/wifi"
+	"github.com/josephroberts/edge-node-manager/radio/zigbee"
+)
+
+type Type struct {
+	Device deviceType.Type
+	Radio  radioType.Type
+}
+
+func (t Type) Scan(name string, timeout time.Duration) ([]string, error) {
```

Comment:
I think it makes much more sense for it to be in the radio type or attached to the radio, since that does the scan.